core/src/main/java/org/hibernate/ogm/dialect/spi/GridDialect.java

```diff
@@ -47,16 +47,17 @@
 	 *
 	 * @param key The tuple identifier
 	 * @param tupleContext Contains additional information that might be used to create the tuple
+	 * @param transactionContext Contains information related to the running transaction, it might be null when the db does not require transacitons
 	 * @return the tuple identified by the key
 	 */
-	Tuple getTuple(EntityKey key, TupleContext tupleContext);
+	Tuple getTuple(EntityKey key, TupleContext tupleContext, TransactionContext transactionContext);
```

Comment:
@gunnarmorling  QueryableGridDialect#execute* does not have a context. Should I add the TransactionCOntext as parameter or should I create a new QueryContext for future extensions?